Backend/OmegaSpring/src/main/java/com/project/omega/exceptions/NoRecordsFoundException.java

```diff
@@ -3,7 +3,7 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
-@ResponseStatus(value = HttpStatus.NO_CONTENT)
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
```

Comment:
No Content is good if it's to signify that the response is empty, not found is usually used if the endpoint can't be found (Error 404)